source/blog/2018-10-26-the-ember-times-issue-70.md

```diff
@@ -12,8 +12,18 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [ember-i18n Deprecation and Release of ember-intl 🌐](https://twitter.com/MiguelCamba/status/1054699605865177089)
+
+[@snewcomer](https://github.com/snewcomer) and [@cibernox](https://github.com/cibernox) [announced](https://twitter.com/MiguelCamba/status/1054699605865177089) a new way to internationalize Ember apps, [ember-i18n](https://github.com/jamesarosen/ember-i18n) is now deprecated in favour of [ember-intl](https://github.com/ember-intl/ember-intl)! This will provide Ember with a standard package for internationalization.
+
+There are [many reasons](https://twitter.com/MiguelCamba/status/1054720978478084097) to make the change such as:
+- Using the [**ICU message format**](https://formatjs.io/guides/message-syntax/) which is a standard in the i18n industry
+- **Locale-aware** numbers, dates, times, currencies, decimals, and percentages!
+- Uses the [**Native Intl API**](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl)
+- Translations are way more powerful and **can contain logic** (like gender-dependent translations)
+- Advanced [**addon support**](https://github.com/ember-intl/ember-intl/blob/master/docs/addon-support.md) to provide translations to the host app
```

Comment:
It seems that [our language linter objects against the use of `host` in this sentence](https://travis-ci.org/emberjs/website/builds/445900488?utm_source=github_status&utm_medium=notification). I think as the word is ok to be used in this context, it might make sense to just escape this particular line [from linting](https://github.com/get-alex/alex#control)

Reply:
Added an ignore!